README.md

```diff
@@ -1,2 +1,143 @@
 # orchestrator-helm-chart
-Helm chart to deploy the Orchestrator solution suite, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+Helm chart to deploy the Orchestrator solution suite on OpenShit, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+
+This chart will deploy the following on the target OpenShift cluster:
+  - Janus IDP backstage
+  - SonataFlow Operator
+  - OpenShift Serverless Operator
+  - Knative Eventing
+  - Knative Serving
+  - Data Index Service
+  - Job Service
+  - PostgreSQL Databases 
+  - Sample workflow (event-timeout)
+
+## Installation
+
+### From source
+```console
+$ git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
+$ cd orchestrator-helm-chart/charts
+$ helm dep update orchestrator
+$ helm install orchestrator orchestrator
+$ helm upgrade orchestrator orchestrator  --set includeCustomResources=true
+```
+Note that you shall first do a `helm install`, wait for the SonataFlow operator and OpenShift Serverless operator to get started,
+and then perform a `helm upgrade with --set includeCustomResources=true` to deploy the remaining components using custom resources.
+  
+Wait and watch for the deployed components to be up and running.
+```console
+$ oc get pod -n openshift-operators | grep sonataflow-operator-controller-manager
+sonataflow-operator-controller-manager-7c6b788cd-2qh8h   2/2     Running   2          12h
+$ oc get pod -n openshift-serverless
+NAME                                        READY   STATUS    RESTARTS   AGE
+knative-openshift-5d95b98764-n4lqt          1/1     Running   1          6h42m
+knative-openshift-ingress-99c7d4f9f-d5zcd   1/1     Running   1          6h42m
+knative-operator-webhook-ffc54cb74-4qtsp    1/1     Running   1          6h42m
+$ oc get pod -n openshift-serverless |grep knative
+knative-openshift-5d95b98764-n4lqt          1/1     Running   1          6h42m
+knative-openshift-ingress-99c7d4f9f-d5zcd   1/1     Running   1          6h42m
+knative-operator-webhook-ffc54cb74-4qtsp    1/1     Running   1          6h42m
+$ oc get pod
+NAME                                               READY   STATUS    RESTARTS   AGE
+janus-idp-orchestrator-backstage-ff99669f5-jq2dx   1/1     Running   3          6h43m
+janus-idp-orchestrator-postgresql-0                1/1     Running   1          6h43m
```

Comment:
Postgres seems to stay in `Pending` for me (and so `orchestrator-backstage` can't move to `Running`).
Can you please double check?

Here some info
```bash
$ oc describe pod orchestrator-postgresql-0
Name:           orchestrator-postgresql-0
Namespace:      default
Priority:       0
Node:           <none>
Labels:         app.kubernetes.io/component=primary
                app.kubernetes.io/instance=orchestrator
                app.kubernetes.io/managed-by=Helm
                app.kubernetes.io/name=postgresql
                controller-revision-hash=orchestrator-postgresql-5c5f6d5dfb
                helm.sh/chart=postgresql-12.5.2
                statefulset.kubernetes.io/pod-name=orchestrator-postgresql-0
Annotations:    <none>
Status:         Pending
IP:             
IPs:            <none>
Controlled By:  StatefulSet/orchestrator-postgresql
Containers:
  postgresql:
    Image:      quay.io/fedora/postgresql-15:latest
    Port:       5432/TCP
    Host Port:  0/TCP
    Requests:
      cpu:      250m
      memory:   256Mi
    Liveness:   exec [/bin/sh -c exec pg_isready -U "bn_backstage" -h 127.0.0.1 -p 5432] delay=30s timeout=5s period=10s #success=1 #failure=6
    Readiness:  exec [/bin/sh -c -e exec pg_isready -U "bn_backstage" -h 127.0.0.1 -p 5432
] delay=5s timeout=5s period=10s #success=1 #failure=6
    Environment:
      BITNAMI_DEBUG:                        false
      POSTGRESQL_PORT_NUMBER:               5432
      POSTGRESQL_VOLUME_DIR:                /var/lib/pgsql/data
      PGDATA:                               /var/lib/pgsql/data/userdata
      POSTGRES_USER:                        bn_backstage
      POSTGRES_POSTGRES_PASSWORD:           <set to the key 'postgres-password' in secret 'orchestrator-postgresql'>  Optional: false
      POSTGRES_PASSWORD:                    <set to the key 'password' in secret 'orchestrator-postgresql'>           Optional: false
      POSTGRESQL_ENABLE_LDAP:               no
      POSTGRESQL_ENABLE_TLS:                no
      POSTGRESQL_LOG_HOSTNAME:              false
      POSTGRESQL_LOG_CONNECTIONS:           false
      POSTGRESQL_LOG_DISCONNECTIONS:        false
      POSTGRESQL_PGAUDIT_LOG_CATALOG:       off
      POSTGRESQL_CLIENT_MIN_MESSAGES:       error
      POSTGRESQL_SHARED_PRELOAD_LIBRARIES:  pgaudit
      POSTGRESQL_ADMIN_PASSWORD:            <set to the key 'postgres-password' in secret 'orchestrator-postgresql'>  Optional: false
    Mounts:
      /dev/shm from dshm (rw)
      /var/lib/pgsql/data from data (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-qh7cs (ro)
Conditions:
  Type           Status
  PodScheduled   False 
Volumes:
  data:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  data-orchestrator-postgresql-0
    ReadOnly:   false
  dshm:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:     Memory
    SizeLimit:  <unset>
  kube-api-access-qh7cs:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
    ConfigMapName:           openshift-service-ca.crt
    ConfigMapOptional:       <nil>
QoS Class:                   Burstable
Node-Selectors:              <none>
Tolerations:                 node.kubernetes.io/memory-pressure:NoSchedule op=Exists
                             node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason            Age                From               Message
  ----     ------            ----               ----               -------
  Warning  FailedScheduling  52m                default-scheduler  0/2 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/2 nodes are available: 2 Preemption is not helpful for scheduling..
  Warning  FailedScheduling  27m (x5 over 47m)  default-scheduler  0/2 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/2 nodes are available: 2 Preemption is not helpful for scheduling..

```
```bash
$ oc get pv 
No resources found
```

```bash
$ oc get pvc
NAME                             STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
data-orchestrator-postgresql-0   Pending                                                     54m
```
```bash
$ oc describe pvc data-orchestrator-postgresql-0
Name:          data-orchestrator-postgresql-0
Namespace:     default
StorageClass:  
Status:        Pending
Volume:        
Labels:        app.kubernetes.io/component=primary
               app.kubernetes.io/instance=orchestrator
               app.kubernetes.io/name=postgresql
Annotations:   <none>
Finalizers:    [kubernetes.io/pvc-protection]
Capacity:      
Access Modes:  
VolumeMode:    Filesystem
Used By:       orchestrator-postgresql-0
Events:
  Type    Reason         Age                    From                         Message
  ----    ------         ----                   ----                         -------
  Normal  FailedBinding  4m33s (x207 over 54m)  persistentvolume-controller  no persistent volumes available for this claim and no storage class is set
```

Reply:
Update: a default storage class is required and I added a "Prerequisites" section.